include/swift/AST/ModuleDependencies.h

```diff
@@ -543,7 +543,18 @@ class ModuleDependencyInfo {
   /// Add a dependency on the given module, if it was not already in the set.
   void addModuleImport(ImportPath::Module module,
                        llvm::StringSet<> *alreadyAddedModules = nullptr) {
-    addModuleImport(module.front().Item.str(), alreadyAddedModules);
+    std::string ImportedModuleName = module.front().Item.str().str();
```

Comment:
Is the `.str().str()` intentional?

Reply:
Sadly, yes. Not enough to have a `StringRef` no that we may be changing the value of this.